### DISCUSSION TITLE
Networking Top Paths module progress display

### DISCUSSION THREAD

**Discussion by Lmmmmmm-bb** [vercel/vercel#9407] [Help] [answered]
The Top Paths module displays the same width (blue part), is this a bug?

![image](https://user-images.githubusercontent.com/54026110/218096659-c0ac764d-5967-4694-a4dd-d3b1f357bb3f.png)

**Comment by leerob** [comment, marked as answer]
Hey! This was a bug, thanks for letting us know. It's now fixed!
